iatidatacube/import_data.py

```diff
@@ -79,29 +81,30 @@ def delete_dataset(csv_file, inserted_ids):
     dataset_type, dataset_country = re.match("(.*)-(.*).csv", csv_file).groups()
     if dataset_type == 'budget':
         q = sa.sql.expression.select(IATILine.id).where(
-            IATILine.recipient_country_or_region==dataset_country,
-            IATILine.transaction_type=='budget'
-        ).order_by(IATILine.id)
+            IATILine.recipient_country_or_region == dataset_country,
+            IATILine.transaction_type == 'budget'
+        ).order_by(IATILine.id)  # sjk: now that 'id' is a hash, its unclear whether this ordering does anything useful
```

Comment:
Oh interesting yes -- I originally did this to speed up searching in the list of identifiers. But as the list of identifiers is now a set, which is unordered, I don't think this `.order_by()` now does anything?